GDJS/Runtime/pixi-renderers/pixi-bitmapfont-manager.ts

```diff
@@ -289,6 +289,18 @@ namespace gdjs {
         );
       }
     }
+
+    dispose(): void {
+        for (const bitmapFontInstallKey in this._pixiBitmapFontsInUse) {
+            PIXI.BitmapFont.uninstall(bitmapFontInstallKey);
+        }
+        for (const bitmapFontInstallKey of this._pixiBitmapFontsToUninstall) {
+            PIXI.BitmapFont.uninstall(bitmapFontInstallKey);
+        }
+        this._pixiBitmapFontsInUse = {};
+        this._pixiBitmapFontsToUninstall = [];
```

Comment:
Might be: `this._pixiBitmapFontsToUninstall.length = 0;`

Reply:
done